packages/safe-tools-client/app/components/schedule-payment-form-action-card/ui/index.gts

```diff
@@ -295,51 +297,66 @@ export default class SchedulePaymentFormActionCardUI extends Component<Signature
           </div>
         </BoxelField>
       </Section>
-        <ActionChin @state={{this.actionChinState}}>
-          <:default as |ac|>
+      <ActionChin @state={{this.actionChinState}}>
+        <:default as |ac|>
+          {{#if @isSafesEmpty}}
             <ac.ActionButton
               @disabled={{not @isSubmitEnabled}}
               data-test-schedule-payment-form-submit-button
               {{on 'click' @onSchedulePayment}}
             >
               Schedule Payment
             </ac.ActionButton>
-          </:default>
-          <:inProgress as |ac|>
-            <ac.ActionStatusArea @icon={{concat @walletProviderId "-logo" }} style={{cssVar status-icon-size="2.5rem"}}>
-              <BoxelLoadingIndicator
-                class="schedule-payment-form-action-card__loading-indicator"
-                @color="var(--boxel-light)"
-              />
-              <div class="schedule-payment-form-action-card__in-progress-message" data-test-in-progress-message>
-                {{@schedulingStatus}}
+          {{else}}
+            <ac.ActionStatusArea style={{cssVar status-icon-size="2.5rem"}}>
+              <div class="scheduled-payment-form-action-card__empty-safe-stage-icon">
```

Comment:
Let's make it so that this markup is reusable for the other case where we will prompt the user to connect the wallet. Consider generalizing classes, something like:

```
<div class="schedule-payment-form-prerequisite-step">
  <div class="schedule-payment-form-prerequisite-step__icons">
    {{svgJar 'icon123' width="..." height="..."
      style=(cssVar icon-color='var(--boxel-teal)')
    }}
    {{svgJar ...}}
    {{svgJar ...}}
  </div>
  <div class="schedule-payment-form-prerequisite-step__text">
    Step 2 - Create a payments safe
  </div>
</div>
```

```
.schedule-payment-form-prerequisite-step {
  display: flex;
}

.schedule-payment-form-prerequisite-step__icons svg {
  margin-left: ...;
}

.schedule-payment-form-prerequisite-step__text {
  margin: auto;
  ... 
}
```

With this approach, we can simply reuse that for the other case ("Step 1 - First connect your wallet"), we can just change the copy and icons.

Reply:
Reused your code for that